src/uwtools/templater.py

```diff
@@ -102,7 +122,8 @@ def set_template(argv):
 
     if user_args.dry_run:
         if user_args.outfile:
-            print(f'warning file {user_args.outfile} not written when using --dry_run')
+            log.info('warning file %(user_args.outfile)s not written when using --dry_run')
```

Comment:
Is there a typo here with the s after `%(user_args.outfile)`?

Reply:
ah, a remnant of an earlier %s formatting version, I'll replace it to be consistent with the other changes